packages/common/src/project/versioned/base.ts

```diff
@@ -36,7 +36,7 @@ export abstract class ProjectManifestBaseImpl<D extends object> {
 }
 
 export interface BaseDataSource<
-  F = Record<string, unknown>,
+  F = any,
```

Comment:
We leave it to `any` type, this is still working, according to https://github.com/microsoft/TypeScript/issues/50087